server_admin/topics/threat/admin.adoc

```diff
@@ -0,0 +1,95 @@
+
+=== Admin Endpoints and Console
+
+{project_name} administrative REST API and web console are by default exposed on the same port as non-admin usage. If you
+are exposing {project_name} on the Internet we recommend not exposing the admin endpoints on the Internet as well.
+
```

Comment:
The {project_name} administrative REST API and the web console are exposed by default on the same port as non-admin usage. If you are exposing {project_name} on the Internet, we recommend not also exposing the admin endpoints on the Internet.